### PR TITLE
Logging Cleanup - Part 4

### PR DESCRIPTION
**Pull request by xWink**
**What type of PR is this?**

cleanup

**Which issue does this PR fix**:
#126 partially

**What does this PR do / Why do we need it**:
- Removes unnecessary logs
- Removes cases of log n throw
- Grammatical improvements to logs
- Removes embedding structs into logs where not absolutely needed
- Removes use of glog entirely
- Change some info logs to error or debug logs

**If an issue # is not available please add repro steps and logs from aws-gateway-controller showing the issue**:

**Testing done on this change**:

```
Ran 32 of 32 Specs in 2099.127 seconds
SUCCESS! -- 32 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestIntegration (2100.16s)
PASS
ok      github.com/aws/aws-application-networking-k8s/test/suites/integration   2100.892s
```

**Automation added to e2e**:

**Will this PR introduce any new dependencies?**:

No

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
No

**Does this PR introduce any user-facing change?**:

Only logging changes

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by xWink** [comment]
There will be 1 more PR after this with additional cleanup, since we have a handful of places left that contain unwanted or ugly info logs, needlessly scary error logs, and embedded structs in info logs creating clutter

File: controllers/serviceexport_controller.go [hunk 1]
```diff
@@ -129,18 +130,17 @@ func (r *ServiceExportReconciler) reconcile(ctx context.Context, req ctrl.Reques
...
 				return err
 			}
-
-			r.log.Info("Successfully delete")
-
-			r.finalizerManager.RemoveFinalizers(ctx, srvExport, serviceExportFinalizer)
+			err := r.finalizerManager.RemoveFinalizers(ctx, srvExport, serviceExportFinalizer)
+			if err != nil {
+				r.log.Errorf("Failed to remove finalizers for service export %s-%s due to %s",
+					srvExport.Name, srvExport.Namespace, err)
```
Comment (scottlaiaws) [inline controllers/serviceexport_controller.go:142]: I think you forgot to return `err` here

Comment (xWink) [reply controllers/serviceexport_controller.go:142]: This didn't previously return the err, and we don't do it in most of our other controllers. I'd rather avoid a behaviour change in a logging-focused PR

Comment (scottlaiaws) [reply controllers/serviceexport_controller.go:142]: oh I mis-read the diff

**Review by mikhail-aws — APPROVED** [review]

**Pull request merged**